### PR TITLE
Feature/csh/exp cluster

### PR DESCRIPTION
**Pull request by cshcomcom**
## Description
- A new cluster environment has been added! (testnet-exp)

### CODE REVIEW

File: tools/genesis-file/createGenesisBlock.js [hunk 1]
```diff
@@ -386,7 +386,7 @@ function getGenesisBlockTxs() {
 function executeGenesisTxsAndGetData(genesisTxs) {
   const tempGenesisDb = new DB(
       new StateNode(StateVersions.EMPTY),
-      StateVersions.EMPTY, null, -1, null, GenesisAccounts.owner.address);
```
Comment (platfowner) [inline tools/genesis-file/createGenesisBlock.js:389]: thanks for fixing this.

File: blockchain-configs/testnet-exp/node_params.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  "MAX_FINALIZED_BLOCK_INFO_ON_MEM": 1000,
+  "MAX_NUM_INBOUND_CONNECTION": 6,
+  "MAX_NUM_SNAPSHOTS": 10,
+  "ON_MEMORY_CHAIN_LENGTH": 10,
+  "P2P_GENESIS_BLOCK_HASH_MISMATCH_SLEEP_MS": 10000,
+  "P2P_HEARTBEAT_INTERVAL_MS": 15000,
+  "P2P_MESSAGE_TIMEOUT_MS": 600000,
+  "P2P_PORT": 5000,
+  "P2P_WAIT_FOR_ADDRESS_TIMEOUT_MS": 10000,
+  "PEER_CANDIDATE_JSON_RPC_URL": "https://exp-api.ainetwork.ai/json-rpc",
```
Comment (platfowner) [inline blockchain-configs/testnet-exp/node_params.json:45]: question: don't we need this domain setup?

**Review by platfowner — COMMENTED** [review]
LGTM, thanks!
Please wait for @liayoo 's approval before merging..

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

File: blockchain-configs/testnet-exp/node_params.json (hunk 2 above)
Comment (cshcomcom) [reply blockchain-configs/testnet-exp/node_params.json:45]: Thanks 👍

**Comment by cshcomcom** [comment]
Thanks for reviews! Now merging

**Pull request merged**